NETOBSERV-2350: Static plugin: feedback follow-up

jpinsonneau opened the pull request:
## Description

- [x] append "..." to truncated text
- [X] auto-generate schema and examples
- [X] 2nd pass on field descriptions, fields to hide, etc.
- [X] code style (camelCase for constants)
- [x] fix linter warnings introduced
- [X] remove < br > tags in descriptions
- [x] add a delete button on edit forms 

## Dependencies

<!-- List here any related PRs with links, that need to be pulled also for testing -->
n/a

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [x] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [x] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [ ] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

jotak commented on web/moduleMapper/dummy.tsx:
```diff
@@ -88,17 +88,79 @@ export function k8sUpdate(k8s: any): Promise<any> {
 }
 
+export function k8sDelete(k8s: any): Promise<any> {
+  console.log("k8sDelete", k8s);
```
is that temporary debug log or something we want to keep?

jpinsonneau replied:
The dummy file is not imported on production builds since it's part of console api. It's useful to see what's been sent in standalone builds.

https://github.com/netobserv/network-observability-console-plugin/blob/main/web/webpack.standalone.js#L81

jpinsonneau commented:
> There seems to be some problems in the wizard:
> 
> * the eBPF features/privilege mode have disappeared? Is it on purpose?
> * Chosing Kafka deployment model, or LokiStack for loki, doesn't trigger an update of the form with the relevant features
> * We seem to have lost the "s/NetObserv/network observability/" replacement. Or maybe this is some text overrides that we have lost? E.g. we had before:
> 
> <img alt="image" width="1038" height="99" src="https://private-user-images.githubusercontent.com/2153442/477950826-e6b20dff-3ff1-4e74-a4be-4eeeb2a8781c.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU1OTAwNzcsIm5iZiI6MTc1NTU4OTc3NywicGF0aCI6Ii8yMTUzNDQyLzQ3Nzk1MDgyNi1lNmIyMGRmZi0zZmYxLTRlNzQtYTRiZS00ZWVlYjJhODc4MWMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDgxOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA4MTlUMDc0OTM3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MzkxZWEyOThhYWVjNDM2MzdkYjU0NDJkM2E4ZThiZDhmOTlmYmIwOTJmYjUxYmNkNGExMGUwZDZkNjk0OWM1NSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.PQBK3bWkpEHcs5dDIiA4f9QCHNOWZyo8lHWbuqJC8ts"> and now: <img alt="image" width="1038" height="99" src="https://private-user-images.githubusercontent.com/2153442/477951075-afcab408-e223-4e00-b266-1ce22adcff4e.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU1OTAwNzcsIm5iZiI6MTc1NTU4OTc3NywicGF0aCI6Ii8yMTUzNDQyLzQ3Nzk1MTA3NS1hZmNhYjQwOC1lMjIzLTRlMDAtYjI2Ni0xY2UyMmFkY2ZmNGUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDgxOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA4MTlUMDc0OTM3WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MjVhOTYzMGUzZWFkNzc3YjRlZTUwNjRiYTk4ZmFkZTFiNDg1Mzk3MDQ0YTU0YzI2MDFjYzk3NjBlYzFkZDVjMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.6BvQ-jeFjJuQSl90X1Iu2xtTxgThjNKvvgf5fD8m9Lg"> The previous text was better
> * In [this doc](https://docs.google.com/document/d/1s4ZKlJaIB7wy8IKlUjXts6WkZgCUDmlyFrM0Zh1fMZE/edit?tab=t.0#heading=h.k67ggqrjj0eo) I suggested some changes of what to add/remove from the wizard, which would make it shorter and more focused on a smaller set of much used features. What's the status, are we ok to do that?

I have double checked this and it's still working. I guess you didn't deployed the operator from OLM. You need a valid CSV deployed to retreive the model:
<img width="1554" height="796" alt="image" src="https://github.com/user-attachments/assets/66b88bfc-00f5-48e5-85b0-475e4deb2a69" />
<img width="1554" height="796" alt="image" src="https://github.com/user-attachments/assets/64aef1a7-5ccb-4149-b0d1-f6bbaf224a41" />

jpinsonneau commented:
Fixed the scroll issue in consumption tab and added some ids
https://github.com/netobserv/network-observability-console-plugin/pull/956/commits/329e78b523e9153b8cf44c9d69c905db2bf2eaec

Amoghrd commented:
Still seeing same issue with latest image. Also flowmetrics form has a bug too wherein it does not get created on 4.15

Amoghrd commented:
Not the double scroll, the browser crashing on trying to reach `flowcollector/status` page.
Using chrome

Amoghrd commented:
On which cases does it crash?

jpinsonneau commented:
@Amoghrd that one will fix the missing name / namespace fields in FlowMetric: https://github.com/netobserv/network-observability-console-plugin/pull/956/commits/a289d61c0bf8f830041971bc3d89442a9b691984

I also improved the way errors are displayed. 

FYI, by default, the FlowMetric example has an issue. A legend field is missing.

OlivierCazade reviewed (approved):
LGTM

Amoghrd commented:
/label qe-approved
